### ISSUE TITLE
Completion should trigger on . after ? in a null-conditional access expression with an implicit With block receiver

### ISSUE THREAD

**Issue by AnthonyDGreen** [dotnet/roslyn#4750]
In the example below completion should pop up after ?. on line 1 (before Length); today it does not.

``` VB.NET
Module Module1
    Sub Main()
        Dim s As String

        With s
1:         Console.WriteLine(If(?.Length, -1))
            Console.WriteLine()
        End With

        Dim dict = New Dictionary(Of String, Object) From {{"Name", "Anthony"}}

        With dict
            Console.WriteLine(?!Name)
        End With
    End Sub
End Module
```
